packages/cli/test/unit/util/build/import-builders.test.ts

```diff
@@ -27,7 +27,67 @@ describe('importBuilders()', () => {
     );
   });
 
-  it('should import 3rd party Builders', async () => {
+  it('should import built-in Builders using `@latest`', async () => {
+    const specs = new Set(['@vercel/node@latest', '@vercel/next@latest']);
+    const builders = await importBuilders(specs, process.cwd(), client.output);
+    expect(builders.size).toEqual(2);
+    expect(builders.get('@vercel/node@latest')?.pkg).toMatchObject(
+      vercelNodePkg
+    );
+    expect(builders.get('@vercel/next@latest')?.pkg).toMatchObject(
+      vercelNextPkg
+    );
+    expect(typeof builders.get('@vercel/node@latest')?.builder.build).toEqual(
+      'function'
+    );
+    expect(typeof builders.get('@vercel/next@latest')?.builder.build).toEqual(
+      'function'
+    );
+  });
```

Comment:
Can we verify that no "npm install" was run?

Perhaps check `client.stderr`?

Reply:
The "not" case depends on a timeout so I prefer not to use that if necessary. I'd rather check the imported file path. Will add a case for that.

Reply:
dc7d3a6b30c3a8d24ca4d8f91c6778f36d0d4877